cli/bin/ci/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -ev
+
+################################
+# Python environment setup
+
+pyenv global 3.6
+
+python --version
+python3 --version
+
+# Explicitly uninstall cli
+if [[ $(pip list --format=columns | grep waiter-client) ]];
+then
+    pip uninstall -y waiter-client
+fi
+
+pip install -e .
+pip install -r integration/requirements.txt
```

Comment:
Do these `pip` commands need to be `pip3`?

Reply:
I think they end up doing `pip3` since we're doing the `pyenv global 3.6` above